frontend/src/pages/add-blog.tsx

```diff
@@ -157,20 +148,32 @@ function AddBlog() {
             ))}
           </div>
         </div>
+        <div className="mt-4 mb-4 ">
+          <textarea
+            name="description"
+            placeholder="Description"
+            className="w-full p-2 rounded-lg bg-gray-50 placeholder:text-gray-800 "
+            value={formData.description}
+            onChange={handleInputChange}
+          />
+        </div>
         <div className="mb-4 flex items-center">
           <label className="flex items-center">
+            <span className="text-gray-800 text-base font-medium">Featured Post</span>
             <input
               type="checkbox"
               name="isFeaturedPost"
-              className="mr-2 h-5 w-5 text-indigo-600 border rounded focus:ring-indigo-400"
+              className="ml-2 h-5 w-5 text-indigo-600 border rounded focus:ring-indigo-400"
               checked={formData.isFeaturedPost}
               onChange={handleCheckboxChange}
             />
-            <span className="text-gray-800 text-base font-medium">Featured Post</span>
           </label>
         </div>
 
-        <button type="submit" className="bg-black text-white p-2 rounded-lg hover:bg-gray-800">
+        <button
+          type="submit"
+          className="bg-black text-white text-[16px] md:text-lg lg:text-xl p-2 rounded-lg hover:bg-gray-800 w-full md:w-fit"
+        >
```

Comment:
try to use equivalent tailwind-css class than hardcoding 16px

Reply:
Ok sir I will make it as text-sm or text-base

Reply:
Sure anything that is close to 16px, please refer the documentation for this